promsonnet/example.jsonnet

```diff
@@ -0,0 +1,24 @@
+local prom = import 'prom.libsonnet';
+local promRuleGroupSet = prom.v1.ruleGroupSet;
+local promRuleGroup = prom.v1.ruleGroup;
+{
+  prometheus_down::
+    promRuleGroup.new('prometheus_down')
```

Comment:
Perhaps name the group different from the rule? For example `prometheus_metamon`.